provider-ui/pages/_app.js

```diff
@@ -229,7 +229,6 @@ class MesheryProviderApp extends App {
           </Head>
           <MuiThemeProvider theme={theme}>
             <div className={classes.root}>
-              <CssBaseline />
```

Comment:
Not required?

Reply:
Provider page is not rendering UI after having "@material-ui/core": "^4.12.4", probable Reason is Client-side Exception Occurred, Basically, Our CssBaseline component is now not compatible with our provider UI. I noticed that the style override via createTheme for CSSBaseline is marked as deprecated. but without including CssBaseline, Our provider UI is rendering UI and working as it should be.

Reply:
Gotcha, thanks for confirming.